postgres-driver/pocket_session_test.go

```diff
@@ -17,7 +17,8 @@ func (ts *PGDriverTestSuite) TestPostgresDriver_WriteSession() {
 			session: types.PocketSession{
 				SessionKey:            "21",
 				SessionHeight:         21,
-				ProtocolApplicationID: 21,
+				ProtocolApplicationID: "21",
```

Comment:
`ProtocolApplicationID` has no meaning, a protocol app only has a public key and address as an identifier, such field should either be renamed to better reflect its value or removed